cmd/logger.go

```diff
@@ -67,9 +67,10 @@ func errorIf(err error, msg string, data ...interface{}) {
 	fields := logrus.Fields{
 		"cause": err.Error(),
 	}
-	if globalTrace {
-		fields["stack"] = "\n" + stackInfo()
+	if e, ok := err.(*Error); ok {
+		fields["stack"] = strings.Join(e.Trace(), " ")
```

Comment:
@harshavardhana it is difficult to put a stack trace (using `stackInfo()`) when it is not available in `err` because it will mean sometimes stack trace represents origin of error and sometimes it represents the place where the errorIf is done - it will be confusing.